cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/containers/NotifyContextRequest.java

```diff
@@ -295,7 +295,7 @@ public String toString() {
     /**
      * Class for storing contextAttribute information from a notifyContextRequest.
      */
-    public class ContextAttribute {
+    public static class ContextAttribute {
```

Comment:
Why this class has changed to static?

Reply:
internal or inner classes could be declared as static.

Reply:
ContextAttribute is an Inner class from NotifyContextRequest, in order to create a new object (for casting) from this class. It was necessary to make it static (since is an inner class)